python/ganga/__init__.py

```diff
@@ -56,6 +56,8 @@ def ganga_license():
 interactive = False
 from Ganga.Core.GangaRepository import getRegistrySlice
 Ganga.Core.bootstrap(getRegistrySlice('jobs'), interactive)
+from Ganga.Runtime.GPIexport import exportToInterface
+exportToInterface(ganga, 'runMonitoring', Ganga.Core.monitoring_component.runMonitoring, 'Functions')
```

Comment:
You should just be able to pass ```ganga``` as an extra argument to ```bootstrap``` above or you end up exporting this to the ```GPI``` as well which is a little bad as this interface shouldn't be used when doing ```import ganga```